compose/install/index.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to install Docker Compose
+keywords: compose, orchestration, install, installation, docker, documentation
+title: Install Docker Compose
+toc_max: 2
+---
+
+This page contains information on how to install Docker Compose. You can run Compose on macOS, Windows, and 64-bit Linux.
+
+
+## Prerequisites
+
+* Docker Compose requires Docker Engine.
+* Docker Compose plugin requires Docker CLI.
+
+## Install Compose
+
+Check what installation scenario fits your needs. Are you looking to:
+
+* __Get latest Compose and pre-requirements of the bat__:
```

Comment:
And I'd make these top-level bullet points, headings instead.

Reply:
Why headings? I'm not putting content under it (and I want content to options to be close to each other). I see this as a manual TOC, sort of a description list: <item>:<description>.

Reply:
I just felt it get's a little lost and provides an entry point for users into the different installation scenarios you have.